packages/standard-minifiers-css/plugin/minify-css.js

```diff
@@ -14,17 +14,19 @@ CssToolsMinifier.prototype.processFilesForBundle = function (files, options) {
 
   if (! files.length) return;
 
-  var merged = mergeCss(files);
-
+  // Don't merge or minify anything for development
   if (mode === 'development') {
-    files[0].addStylesheet({
-      data: merged.code,
-      sourceMap: merged.sourceMap,
-      path: 'merged-stylesheets.css'
+    files.forEach(function (file) {
+      file.addStylesheet({
+        data: file.getContentsAsBuffer(),
+        sourceMap: file.getSourceMap(),
+        path: file.getPathInBundle()
```

Comment:
This is problematic as the path from which the CSS file is served will be different in development and in production. This means that a working `url(...)` in development will break in production mode.

I think this problem would be easily solved by serving all CSS files from the root of the application. Then assuming that the CSS file order is deterministic, the merge minfication shouldn't have any side effect (since properties are loaded in the same order from the same path).